cli/src/commands/extensions/api.rs

```diff
@@ -251,6 +251,10 @@ async fn get_access_token(
 #[op2(async)]
 #[serde]
 async fn get_refresh_token(op_state: Rc<RefCell<OpState>>) -> Result<RefreshToken> {
+    get_refresh_token_impl(op_state).await
+}
+
+async fn get_refresh_token_impl(op_state: Rc<RefCell<OpState>>) -> Result<RefreshToken> {
```

Comment:
There should probably be a comment explaining why this needs to be duplicated. (because `#[op2(async)]` replaces the function with a function that describes the function instead of doing what it's written to do?)

Reply:
I decided to refactor this a bit instead and just implement a `refresh_token()` helper function. See b534da3

It's a bit more efficient this way.